common/lib/xmodule/xmodule/capa_module.py

```diff
@@ -98,8 +98,8 @@ def handle_ajax(self, dispatch, data):
         try:
             result = handlers[dispatch](data)
 
-        except NotFoundError as err:
-            log.exception(
+        except NotFoundError:
+            log.info(
                 "Unable to find data when dispatching %s to %s for user %s",
```

Comment:
We get this error here in the case that either a student tries to submit a problem after it's due or the course team changes something about a problem between the time the student loads the page and they click a button like "show answer": https://splunk.edx.org/en-US/app/search/search?sid=1467899367.558779